app/src/ereferrals/java/org/eyeseetea/malariacare/domain/boundary/repositories/ICurrentLanguageRepository.java

```diff
@@ -0,0 +1,7 @@
+package org.eyeseetea.malariacare.domain.boundary.repositories;
+
+import org.eyeseetea.malariacare.data.IDataSourceCallback;
+
+public interface ICurrentLanguageRepository {
```

Comment:
Repositories should return domain entities, this Repository returns a primitive. Here it may make sense a Settings domain entity with CurrenLanguage getter in this moment and refactor this repository to ISettingsRepository